psh/test-touch-rootfs.py

```diff
@@ -0,0 +1,32 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh touch command test for rootfs targets
+#
+# Copyright 2022 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+
+import psh.tools.psh as psh
+from psh.tools.common import assert_timestamp_change
+
+
+def assert_hardlinks(p):
+    uptimes = {}
+    # all psh commands are hardlinks
+    psh_cmds = psh.get_commands(p)
+    for psh_cmd in psh_cmds:
```

Comment:
You're talking about files (hardlinks) here, not about commands - please change the variable names accordingly